config/application.rb

```diff
@@ -74,7 +74,7 @@ class Application < Rails::Application
 
     # Store boolean values are in sqlite3 databases as 1 and 0 instead of 't' and
     # 'f' after migrating old data.
-    config.active_record.sqlite3.represent_boolean_as_integer = true
+    # config.active_record.sqlite3.represent_boolean_as_integer = true
```

Comment:
is this still needed in some situations (like local development)? Why keep it?

Reply:
No it is not needed. I forgot to remove it, I kept it while testing thinking it was being used.